README.md

```diff
@@ -1,564 +1,126 @@
-           prior to Anaconda 3.4 - source activate isis
-
-
-### Installation with Docker
-The ISIS production Dockerfile automates the conda installation process above.
-You can either build the Dockerfile yourself or use the
-[usgsastro/isis](https://hub.docker.com/repository/docker/usgsastro/isis)
-image from DockerHub.
-
-#### To build the Dockerfile
-1. Download [the production Docker file](./docker/production.dockerfile)
-2. Build the Dockerfile
-  ```
-  docker build -t isis -f production.dockerfile .
-  ```
-3. Run the Dockerfile
-  ```
-  docker run -it isis bash
-  ```
-
-#### Run run the prebuilt image
-```
-docker run -it usgsastro/isis bash
-```
-
-#### Usage with the ISIS data area
-Usually you'll want to mount an external directory containing the ISIS data.
-The data is not included in the Docker image.
-
-```
-docker run -v /my/data/dir:/opt/conda/data -v /my/testdata/dir:/opt/conda/testData -it usgsastro/isis bash
-```
-
-Then [download the data](#the-isis-data-area) into /my/data/dir to make it accessible inside your
-container.
-
-### Practical Usage with other conda packages
+For more info, see:
 
-If you don't use conda for anything else on your computer, you can
-skip this section.
+- [Tutorials and Guides](https://astrogeology.usgs.gov/docs/)
+- [FAQ](https://astrogeology.usgs.gov/docs/getting-started/using-isis-first-steps/isis-faq/)
+- [ISIS Application Manuals](https://isis.astrogeology.usgs.gov)
 
-If you use conda to install other packages, you may run into
-difficulties with adding the isis conda package to those environments
-or adding other conda packages to the isis environment you just
-created above.  This is because the isis conda package pins a number
-of requirements that may clash with other packages.
+For Development and Contributing, see:
 
-At this time, we recommend creating the isis environment as detailed
-above, and then not adding any other conda packages to it.  This
-is similar to the best practice usage of not adding any conda
-packages to your 'base' conda environment.
+- [Building ISIS](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/developing-isis3-with-cmake/)
+- [ISIS Test Data](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/obtaining-maintaining-submitting-test-data/)
+- [Contributing](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/contributing-to-isis/)
 
-Instead, when you need to have a conda environment with other
-packages that also needs to be able to run ISIS programs, we have
-two different options.  In both cases, we'll assume that you create
-a new environment called 'working' (but it could be named anything)
-that you want to add some conda packages to, but from which you
-also want ISIS access.
+## In this README:
 
-The first step is to create 'working' and add whatever conda packages you want.
+- [Requests for Comment](README.md#Requests-for-Comment)
+- [Citing ISIS](README.md#Citing-ISIS)
```

Comment:
These links are a little awkward since they link to the README in the repo (therefore loading a new page) rather than navigating to that section on the same page. iirc`(#Citing-ISIS)` should work?